src/ctelnet.cpp

```diff
@@ -1550,8 +1560,14 @@ void cTelnet::loadReplay(QString& name)
         postMessage(msg);
         replayStream.setDevice(&replayFile);
         loadingReplay = true;
-        mudlet::self()->replayStart();
-        _loadReplay();
+        if (mudlet::self()->replayStart()) {
+            _loadReplay();
+        } else {
+            loadingReplay = false;
+            QString msg = tr("[ WARN ]  - Cannot perform replay, another one may already be in progress,\n"
+                             "try again when it has finished...!");
+            postMessage(msg);
```

Comment:
This leaks a message to the main console when you use `loadRawFile`, it should instead return nil+error.

`...!` is strange, just a `.` will work.

Reply:
That is a side effect of not spotting that replays could originate in the lua sub-system.